pkgs/os-specific/linux/lenovo-legion/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, fetchurl, stdenv, kernel, bash, pkgs }:
+
+stdenv.mkDerivation {
+  pname = "lenovo-legion";
+  inherit (pkgs.lenovo-legion) version src;
+
+  hardeningDisable = [ "pic" ];
+
+  preConfigure = ''
+    sed -i -e '/depmod/d' ./kernel_module/Makefile
+  '';
+
+  makeFlags = kernel.makeFlags ++ [
+    "SHELL=${bash}/bin/bash"
```

Comment:
This is the bash for the host, so this won't cross compile.  Is there any reason we have to use an absolute path, and not just `SHELL=bash`?

Reply:
Just setting it to `bash` worked